content/02.main-text.md

```diff
@@ -317,6 +321,34 @@ While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
 
+## Future Manubot Enhancements
+
+Manubot is still under active development, and we envision major changes in its design and dependencies going forward.
+Currently, manuscript repositories must contain a large number of files that do not directly contain manuscript content.
+While this enables a high-degree of customization, it also increases complexity.
+Therefore, we are investigating whether configuration files with sensible defaults could enable bare-bones repositories that contain manuscript content and little else.
+
+In addition, to simpler usage, we're also looking into simplified setup.
+Presently, setup is complex because users must do advanced command-line operations to clone the `manubot/rootstock` repository and configure Travis CI.
+While we provide detailed instructions, users often struggle replicating the long list of setup commands.
+One priority will be to automate setup to a higher degree.
+However, this may require switching the services Manubot uses for continuous integration (e.g. from Travis CI to GitHub Actions, CircleCI, Drone, or GitLab CI), environment management (e.g. from Conda to Docker), and repository hosting (e.g. from GitHub to GitLab).
+In addition to simplifying setup, such migrations may also present the oppertunity to decrease dependency on propreitary services and address other Manubot shortcomings, such as the current inability to view rendered manuscripts produced by pull request builds.
+
+Upgrading a Manubot instance is an opt-in procedure.
+Therefore, when we introduce fundamental changes, existing manuscripts continue to function.
+However, large Rootstock changes can make upgrading existing manuscripts difficult.
+We are happy to provide users _pro bono_ assistance to upgrade manuscripts.
```

Comment:
Let's tell readers how to do this.  Presumably they should open an issue at `manubot/rootstock`?  Should we say "... upgrade or troubleshoot manuscripts."?